freqtrade/rpc/telegram.py

```diff
@@ -652,9 +694,6 @@ async def _status_msg(self, update: Update, context: CallbackContext) -> None:
                         "*Open Order:* `{open_orders}`"
                         + ("- `{exit_order_status}`" if r['exit_order_status'] else ""))
 
-            lines_detail = self._prepare_order_details(
-                r['orders'], r['quote_currency'], r['is_open'])
-            lines.extend(lines_detail if lines_detail else "")
```

Comment:
looks great! :+1: 

The only question i asked myself while playing with it is - should the regular "status" message include a hint about "you can get order details via `/order <orderid>` - basically "promoting" this command ...
i'm not sure i'd otherwise realize that there's a command with more details ...